ui/app/templates/vault/cluster.hbs

```diff
@@ -12,48 +12,52 @@
             <NamespacePicker @class="navbar-item" @namespace={{this.namespaceQueryParam}} />
           </li>
         {{/if}}
-        <li class={{if (is-active-route "vault.cluster.secrets") "is-active"}}>
+        <li class={{if (is-active-route "vault.cluster.secrets") "is-active"}} role="button" {{on "click" Nav.closeDrawer}}>
```

Comment:
I think I like this pattern better, where we add the "side effects" click event on the event outside the <LinkTo> element, which just has the `@route` link. I'm concerned that making these links technically buttons violates some [A11y concerns](https://www.a11y-101.com/design/button-vs-link), except of course in cases when the button is triggering a save or submit, and then redirecting.

Reply:
I wondered about that too and ultimately followed the suggestions from the linter. I think since the `li` is the parent element it's not actually the link so the role of button might be ok? Basically the li is acting as a button to close the nav drawer. The nested link functions as a link to transition routes.